Average Minutemen Facility Redux

Latentish opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
![2023 05 16-21 52 58](https://github.com/shiptest-ss13/Shiptest/assets/55075747/868baf28-f9c9-471a-a9b3-daa3600ed949)
![2023 05 16-21 53 03](https://github.com/shiptest-ss13/Shiptest/assets/55075747/6e8d7a13-ae7e-44bc-abaf-fbd66c59fb8b)
![cmmbase](https://github.com/shiptest-ss13/Shiptest/assets/55075747/6e950be4-bb5f-47f7-a09f-50b9f748799b)
Adds the Colonial Minutemen Xenohazard Research Compound.
This is a beach ruin with a high combat challenge, and good loot.
-Xeno egg
-Handful of minutemen guns
-The pussy wagon
-A tank
-Some puzzle shenanigans
-LIT CONCRETE!!!!!!
-Beach concrete (lit)
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
We need more beach ruins, and players enjoy large high combat challenge ruins. Plus there's not really another way to get a xeno egg other than one exceedingly rare and not that great lavaland ruin (which should get neutered a bit imo).
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Adds the Xenolab beach ruin
add: Adds /lit and /lit/beach variants for all concrete floors.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Latentish committed:
Introducing the Xenolab!

- Adds the Minutemen Research Compound beach ruin
- Adds lit versions for the conk creet turfs

Latentish committed:
I HATE CONCRETE/LIT

-Adds 2 concrete subtypes for the open turfs, /lit and /lit/beach (because we have to apparently)

Latentish committed:
Minor loot change

-Removes the particle acceleration rifle
-Moves loot around in light of resin nests not holding items stacked on them.

Erikafox commented on _maps/map_catalogue.txt:
```diff
@@ -499,6 +499,10 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Tags = "Medium Combat Challenge", "Medium Loot", "Liveable"
 
+		File Name = "_maps\RandomRuins\BeachRuins\beach_xenolab.dmm"
+		Size = (x = 80)(y = 70)(z = 1)
```
X should be no greater than 75 otherwise we get _extra_ ruin smushing from ships

Latentish replied:
Done.

Latentish committed:
Merge remote-tracking branch 'upstream/master' into Average-Minutemen-Facility-2

Latentish committed:
Various patchie

-Reduces x value of map to 74
-Replaces shrapnel mines with explosive ones
-Reinforces the tank garage
-Adds indestructible hexacrete wrapped around the xenlab and armory

Erikafox reviewed (changes requested):
I'm gonna start at the bottom for this
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/1fb76fba-23ad-42a7-a898-e5f486d20b1e)
There's absolutely no fluff structurage here, making it incredibly obvious that it's a ruin
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/ea4a4cf0-0a25-4bd5-b3da-88fc846d6544)
This does not feel like a believable structure to me. The only way in and out is through the front gate, there's no electrical, there's no atmospherics (not *required* for a beach world, but for a completely sealed building? The mines surrounding it would make it comically hard to access in a safe manner, it's cheesable with 3 bricks of c4 and a pickaxe, or fuck, a pot water grenade. The only decalling in the room is the minutemen star under the tank, there's just a hand grenade and side arm, and a box of stock parts?
Along with double layered blasts which I generally hold to be a mapping sin. If you want it to be harder to get into and stay sealed, set up a proper airlock. Perhaps look into military motorpools for inspiration. 
![StrongDMM_VeddMJ7cMP](https://github.com/shiptest-ss13/Shiptest/assets/94164348/021d9ac8-c8fe-496d-879b-080ea5e1ff5a)
The dirt suffers the same issue in that it's a monotile, no variation byond the fact that sometimes, there are mines on it. 
![StrongDMM_HkrIkdizuB](https://github.com/shiptest-ss13/Shiptest/assets/94164348/6abdc5f2-66af-4082-b242-52305ccb1dcd)
I have no idea what this is supposed to be. A waiting room? A security checkpoint? The decals on the lower one don't *feel* right to me either
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/44e9d5b9-3652-4dc6-b7c9-c6184231dd89)
utterly decalless room, with the main attraction being weirdly spread apart. Consider making it seem obvious that someone was fucking booking it (or died) when they wrote it. Ditch some of the wood or make it damaged / less thick,
![StrongDMM_8cgc6BkZ5X](https://github.com/shiptest-ss13/Shiptest/assets/94164348/02409f70-38c9-4693-a043-9f6350595877)
All of your maints are lightless.
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/2da0ab13-32df-4ac3-9207-733ba4abc81f)
you can use a 2x maint loot spawner there, and also spread the stuff out
![StrongDMM_2RFgRtJN30](https://github.com/shiptest-ss13/Shiptest/assets/94164348/ce3eb00a-fbb4-444f-b6a3-35eb4fafbcdc)
..why is there just a knife?
![StrongDMM_TrRFqVuJ8k](https://github.com/shiptest-ss13/Shiptest/assets/94164348/85bd1c78-7416-4417-a557-5e50d8f4f316)
This APC is completely unwired. As a matter of fact, the entire base is lacking in wires. Everywhere.
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/df1b7874-a9d6-4763-9e63-59fed9a789ae)
Which is strange because you have an engineering area (With a SMES un-terminaled and atop a door). The APC in there will control power for the entire ruin btw.
![StrongDMM_QW6kHFHEgE](https://github.com/shiptest-ss13/Shiptest/assets/94164348/b4de0c62-b11b-4446-b943-69add2cd050a)
None of your random junk is pixelshifted.
Every Single Piece Of Ammo you have placed is not spent.
![StrongDMM_8qmFdvkqks](https://github.com/shiptest-ss13/Shiptest/assets/94164348/c4e5e9e2-2301-47ef-bbdd-c92da77773ea)
why don't you use reinforced walls instead of snowflake concrete. It'd still be tougher to get into, but it lets players use that sandbox element 
![StrongDMM_Nm1fGYkp5Y](https://github.com/shiptest-ss13/Shiptest/assets/94164348/98be1046-804a-4b95-85b5-071a5646deb7)
there are damaged icon states for wooden flooring
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/b37ae9d7-0e17-40d7-bff8-8142627a95c3)
none of your corpses spawn with clothing
![StrongDMM_dqXFOMVVkX](https://github.com/shiptest-ss13/Shiptest/assets/94164348/7cabcdcd-4fbc-4cfe-a2a2-2160980f1c1e)
under ALL of your xeno resin (which players will clear out) is monopattern, undecalled floors. They need to have some decals put under them
![StrongDMM_8ZMl4Qmhsy](https://github.com/shiptest-ss13/Shiptest/assets/94164348/9cf5cff6-3240-4e74-9b9f-18be90c61690)
I would rather this be airlocks with blast doors. It'll feel less gamey. 
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/7058cf8d-a7c6-4db7-a1b5-f850c2259f1b)
This is the wrong type of 5.56. It's for solgov weapons. See if you can bully apogee into spriting 5.56 and whatever the AK calibre is ammo boxes. 
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/7f2fbd3d-fd06-4aed-8a6c-e351af29898d)
use slab tiles here. 
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/23368cdb-bd6e-4fb9-b0a5-d8658b7b17d2)
This area needs Way More debris
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/d4e6eafa-b28d-46be-ba87-125da60ab1d7)
use pavement tiles maybe?
Overall: Needs way more decal work, needs pixel shifting and other fluffing. Should probably be redesigned to mimic a real structure more because it feels less like a ruin and more like a dungeon.

Latentish commented:
This would basically need to be made again from the ground up.
